solvers/BMI/util/forwardKinPos.m

```diff
@@ -0,0 +1,5 @@
+function pos2 = forwardKinPos(q,r)
```

Comment:
document this. and it probably needs a name that makes it more distinct from our main kinematics code.